cmd/fluxd/main.go

```diff
@@ -345,8 +347,27 @@ func main() {
 	}
 
 	// Import GPG keys, if we've been told where to look for them
+
+	if err != nil {
```

Comment:
What does this log?

Reply:
Woops, I was reorganizing my changes and must have included this twice 😄